### DISCUSSION TITLE
Track time to loong

### DISCUSSION THREAD

**Discussion by Lorenzuou** [mikel-brostrom/yolo_tracking#1132] [Q&A]
I have a tracker + yolo_nas setup, at first the predict function from yolo_nas super_gradients was to slow, so i changed it and now it runs at 30ms

but the tracker is running at 100ms-300ms, all deepSort, strongSort and botSort are doing the same time in average. This speed makes my stream application to slow

how can i speed up the tracking? (yes, i am using at gpu enviroment)

**Comment by mikel-brostrom** [comment]
You could use bytetrack or ocsort instead, which are ReID model free tracking methods

**Reply by Lorenzuou** [reply]
Yes, but, is the latency I mentioned for tracking methods with models that slow? I am using a 3060 to run it with 640x640 images

**Comment by mikel-brostrom** [comment]
`yolo_nas` is SLOW. Their inference methods have large overheads. For some reason they don't intend to fix this.

**Reply by Lorenzuou** [reply]
I change it to get the raw output from usual inference and it got better, funny thing is they put the latency ate their page at 7 ms

**Reply by mikel-brostrom** [reply]
I thinks this is only for their quantized onnx models. Their `.pt` inference is poorly optimized
